### PR TITLE
Add a basic FAQ doc + 1 entry

### PR DESCRIPTION
**Pull request by yuvipanda**
We get lots of questions that can be answered here!

### CODE REVIEW

**Comment by betatim** [comment]
Any idea why travis would be unhappy about more docs?

File: docs/source/faq.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Frequently Asked Questions (FAQ)
+
+A collection of frequently asked questions with answers!
+
+If you have a question & have found an answer, send a PR to add it here!
+
+## Can I use repo2docker to bootstrap my own Dockerfile?
+
+If you pass the `--debug` flag to `repo2docker`, it outputs the intermediate
```
Comment (betatim) [inline docs/source/faq.md]: Lead with the answer: "No you can't."

Comment (betatim) [reply docs/source/faq.md]: Should we be consistent and use `jupyter-repo2docker` when referring to the command line program?

File: docs/source/faq.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+Dockerfile it is using to build the docker image. While it is tempting to copy
```
Comment (betatim) [inline docs/source/faq.md]: "Dockerfile it is using" -> "Dockerfile that is used to build"?

File: docs/source/faq.md (hunk 1 above)
Comment (yuvipanda) [reply docs/source/faq.md]: @betatim https://github.com/jupyter/repo2docker/pull/121 is the fix for the jupyter-repo2docker / repo2docker inconsistentcy I think.

**Comment by yuvipanda** [comment]
@betatim just seem to be travis / docker issues, unrelated to our code

**Comment by choldgraf** [comment]
+1 on the page. The subsections will be placed to the left so it should be easy for people to scan the page if it ever gets big...though at some point in the future I'd be +1 on having collapsible Q/A sections

**Comment by betatim** [comment]
LGTM

**Pull request merged**